plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -166,4 +169,23 @@ public String toString()
     {
         return getId() + ":" + getName() + ":" + type.getDisplayName();
     }
+
+    public static IcebergColumnHandle getPathColumnHandle(int id)
+    {
+        return new IcebergColumnHandle(
```

Comment:
in hindsight of adding new metadata column handles, you could create a utility method that creates a `ColumnIdentity` out of a `IcebergMetadataColumn`

Reply:
@findinpath add the util method to build `ColumnIdentity` as well, thanks.